dynamics/src/DGTransport.cpp

```diff
@@ -10,170 +10,152 @@
 {
-    return Eigen::Matrix<Nextsim::FloatType, 1, 3>(cv(eid, 0) + 0.5 * cv(eid, 2) + 1. / 6. * cv(eid, 4),
-        cv(eid, 1) + 0.5 * cv(eid, 5), cv(eid, 3));
+    return Eigen::Matrix<Nextsim::FloatType, 1, 3>(
+        cv(eid, 0) + 0.5 * cv(eid, 2) + 1. / 6. * cv(eid, 4), cv(eid, 1) + 0.5 * cv(eid, 5),
+        cv(eid, 3));
 }
 
 // dG2+ (8 in cell, 3 on edge)
 template <>
-Eigen::Matrix<Nextsim::FloatType, 1, 3>
-leftedgeofcell(const DGVector<8>& cv, size_t eid)
+Eigen::Matrix<Nextsim::FloatType, 1, 3> leftedgeofcell(const DGVector<8>& cv, size_t eid)
 {
-    return Eigen::Matrix<Nextsim::FloatType, 1, 3>(cv(eid, 0) - 0.5 * cv(eid, 1) + 1. / 6. * cv(eid, 3),
-						   cv(eid, 2) - 0.5 * cv(eid, 5) + 1./6. * cv(eid,6),
-						   cv(eid, 4) - 0.5 * cv(eid,7));
+    return Eigen::Matrix<Nextsim::FloatType, 1, 3>(
+        cv(eid, 0) - 0.5 * cv(eid, 1) + 1. / 6. * cv(eid, 3),
+        cv(eid, 2) - 0.5 * cv(eid, 5) + 1. / 6. * cv(eid, 6), cv(eid, 4) - 0.5 * cv(eid, 7));
 }
 template <>
-Eigen::Matrix<Nextsim::FloatType, 1, 3>
-rightedgeofcell(const DGVector<8>& cv, size_t eid)
+Eigen::Matrix<Nextsim::FloatType, 1, 3> rightedgeofcell(const DGVector<8>& cv, size_t eid)
 {
-  return Eigen::Matrix<Nextsim::FloatType, 1, 3>(cv(eid, 0) + 0.5 * cv(eid, 1) + 1. / 6. * cv(eid, 3),
-						 cv(eid, 2) + 0.5 * cv(eid, 5) + 1. / 6. * cv(eid, 6),
-						 cv(eid, 4) + 0.5 * cv(eid, 7));
+    return Eigen::Matrix<Nextsim::FloatType, 1, 3>(
+        cv(eid, 0) + 0.5 * cv(eid, 1) + 1. / 6. * cv(eid, 3),
+        cv(eid, 2) + 0.5 * cv(eid, 5) + 1. / 6. * cv(eid, 6), cv(eid, 4) + 0.5 * cv(eid, 7));
 }
 template <>
-Eigen::Matrix<Nextsim::FloatType, 1, 3>
-bottomedgeofcell(const DGVector<8>& cv, size_t eid)
+Eigen::Matrix<Nextsim::FloatType, 1, 3> bottomedgeofcell(const DGVector<8>& cv, size_t eid)
 {
-  return Eigen::Matrix<Nextsim::FloatType, 1, 3>(cv(eid, 0) - 0.5 * cv(eid, 2) + 1. / 6. * cv(eid, 4),
-						 cv(eid, 1) - 0.5 * cv(eid, 5) + 1. / 6. * cv(eid, 7), 
-						 cv(eid, 3) - 0.5 * cv(eid, 6));
+    return Eigen::Matrix<Nextsim::FloatType, 1, 3>(
+        cv(eid, 0) - 0.5 * cv(eid, 2) + 1. / 6. * cv(eid, 4),
+        cv(eid, 1) - 0.5 * cv(eid, 5) + 1. / 6. * cv(eid, 7), cv(eid, 3) - 0.5 * cv(eid, 6));
 }
-  template <>
-Eigen::Matrix<Nextsim::FloatType, 1, 3>
-topedgeofcell(const DGVector<8>& cv, size_t eid) 
+template <> Eigen::Matrix<Nextsim::FloatType, 1, 3> topedgeofcell(const DGVector<8>& cv, size_t eid)
 {
-  return Eigen::Matrix<Nextsim::FloatType, 1, 3>(cv(eid, 0) + 0.5 * cv(eid, 2) + 1. / 6. * cv(eid, 4),
-						 cv(eid, 1) + 0.5 * cv(eid, 5) + 1. / 6. * cv(eid, 7),
-						 cv(eid, 3) + 0.5 * cv(eid, 6));
+    return Eigen::Matrix<Nextsim::FloatType, 1, 3>(
+        cv(eid, 0) + 0.5 * cv(eid, 2) + 1. / 6. * cv(eid, 4),
+        cv(eid, 1) + 0.5 * cv(eid, 5) + 1. / 6. * cv(eid, 7), cv(eid, 3) + 0.5 * cv(eid, 6));
 }
 
 // %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
If we are fixing the format, do we want to get rid of these section breaks? They aren't used anywhere else in the model.